lib/language.js

```diff
@@ -70,8 +73,9 @@ function init() {
       ,dk: 'Tir'
       ,fi: 'Ti'
       ,nb: 'Tir'
+      ,he: 'ג'
       },
-    ',We' : {
+    ,'We' : {
       cs: 'St'
       ,de: 'Mi'
       ,es: 'Mie'
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected an assignment or function call and instead saw an expression.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/982736897/issues/source?bid=2452379&fileBranchId=2865971#l81)